docs/lib/predictions/fragments/ios/identify-text.md

```diff
@@ -35,6 +54,29 @@ func detectText(_ image: URL) {
 }
 ```
 
+To get results that utilize on-device capabilities (Core ML), without combining results from the backend, you can use the following to pass into the `options` argument of the `Amplify.Predictions.identify` function.
+```swift
+let options = PredictionsIdentifyRequest.Options(defaultNetworkPolicy: .offline, pluginOptions: nil)
+```
+
+Note that bounding boxes in `IdentifyTextResult` are returned as ratios.  If you would like to place bounding boxes on individual recognized words that appear in the image, use the following method to calculate a frame for a single bounding box.
+```swift
+let image = UIImage(...)
```

Comment:
what's the expectation here instead of `...`?  maybe to be a bit more explicit `UIImage(NAME_OF_YOUR_ASSET_FILE)`